apps/server/src/modules/board/service/submission-item.service.ts

```diff
@@ -37,4 +40,15 @@ export class SubmissionItemService {
 
 		return submissionItem;
 	}
+
+	async update(submissionItem: SubmissionItem, completed: boolean): Promise<void> {
+		const parent = (await this.boardDoRepo.findParentOfId(submissionItem.id)) as SubmissionContainerElement;
+		const now = new Date();
+		if (parent.dueDate && parent.dueDate < now) {
+			throw new ValidationError('not allowed to save anymore');
```

Comment:
throw BusinessError instead of ValidationError

Reply:
ValidationError extends BusinessError.
BusinessError is the base abstract class. so this is correct.